R/stash.R

```diff
@@ -155,10 +162,15 @@ write_val <- function(var, val) {
 
 
 # Load in a variable from disk and assign it to the global environment.
-load_variable <- function(var) {
+load_variable <- function(var, functional) {
   path <- stash_filename(var)$data_name
   val <- qs::qread(path)
-  assign_value(var, val)
+  if(functional) {
+    val
+  } else {
+    assign_value(var, val)
+    NULL
+  }
```

Comment:
Can you add explicit return calls? I think that would make the code a bit clearer and could potentially prevent some unexpected behavior (especially if new features are added).